packages/text-editor/src/components/ReferenceInput.svelte

```diff
@@ -48,6 +48,7 @@
   export let placeholder: IntlString | undefined = undefined
   export let extraActions: RefAction[] | undefined = undefined
   export let loading: boolean = false
+  export let basicInput: boolean = false
```

Comment:
I do not like this solution, better to extract actions retrieval outside and pass a list of of actions suitable or not pass.